legend-pure-m3-core/src/main/resources/platform/pure/m3.pure

```diff
@@ -3139,6 +3139,14 @@
                 Root.children[meta].children[pure].children[metamodel].children[type].children[Any].properties[classifierGenericType] : ^Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType]{Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType].properties[rawType]:Root.children[meta].children[pure].children[metamodel].children[function].children[property].children[Property], Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType].properties[typeArguments]:[^Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType]{Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType].properties[rawType]:Root.children[meta].children[pure].children[test].children[Test]}, ^Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType]{Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType].properties[rawType]:String}], Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType].properties[multiplicityArguments]:Root.children[meta].children[pure].children[metamodel].children[multiplicity].children[PureOne]}
+            },
+            ^Root.children[meta].children[pure].children[metamodel].children[function].children[property].children[Property] testable
+            {
+                Root.children[meta].children[pure].children[metamodel].children[ModelElement].properties[name] : 'testable',
+                Root.children[meta].children[pure].children[metamodel].children[function].children[property].children[Property].properties[aggregation] : Root.children[meta].children[pure].children[metamodel].children[function].children[property].children[AggregationKind].values[None],
+                Root.children[meta].children[pure].children[metamodel].children[function].children[property].children[AbstractProperty].properties[genericType] : ^Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType]{Root.children[meta].children[pure].children[metamodel].children[type].children[generics].children[GenericType].properties[rawType]:Root.children[meta].children[pure].children[test].children[Testable]},
+                Root.children[meta].children[pure].children[metamodel].children[function].children[property].children[AbstractProperty].properties[multiplicity] : Root.children[meta].children[pure].children[metamodel].children[multiplicity].children[PureOne],
```

Comment:
I notice this has multiplicity 1, meaning it's not backward compatible. Is that something we need to worry about?

Reply:
Is this intended to be the reverse of the tests property on Testable? If so, should the relationship be modeled with an association instead of as two separate properties?

Reply:
@kevin-m-knight-gs yes its expected to be non backward compatible. Yes this is meant to be an association though not sure what you mean exactly as we have Association in m3 level. Can you help me understand how to better model the relationship